todocore/src/test/java/com/knighttodo/todocore/integration/DayTodoResourceIntegrationTest.java

```diff
@@ -331,5 +340,22 @@ public void findDayTodosByDayId_shouldReturnExistingTodo_whenIdIsCorrect() throw
                 .andExpect(status().isFound())
                 .andExpect(jsonPath(buildJsonPathToLength()).value(2));
     }
+    //my test
+    @Test
+    public void updateOrderNumberByDayTodoId_shouldReturnUpdatedTodo_whenAllDayIdTodoExist() throws Exception {
+        Day day = dayRepository.save(DayFactory.dayInstance());
+        DayTodo firstDayTodo = dayTodoRepository.save(DayTodoFactory.dayTodoWithDayInstance(day));
+        dayTodoRepository.save(DayTodoFactory.dayTodoWithDayInstance(day));
+
+        Map<UUID, Integer> map = new HashMap<>();
+        map.put(firstDayTodo.getId(), 777);
+
+        mockMvc.perform(patch(API_BASE_URL_V1 + API_BASE_DAYS + "/" + day.getId() + API_BASE_TODOS)
+                .content(objectMapper.writeValueAsString(map))
+                .contentType(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
check please if you return the updated order number in json response

Reply:
Done.